From ebcc8117f7642520a57dc9a5dc3dd8734c211d49 Mon Sep 17 00:00:00 2001
From: Ola Jeppsson <ola@adapteva.com>
Date: Sun, 24 Mar 2019 19:56:51 -0400
Subject: [PATCH 33/33] epiphany: Fix fsoftware-cache description

gcc/
	* config/epiphany/epiphany.opt (fsoftware-cache): Add missing
	dot in description. Fixes a testcase.
---
 gcc/ChangeLog.epiphany           | 5 +++++
 gcc/config/epiphany/epiphany.opt | 2 +-
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/gcc/ChangeLog.epiphany b/gcc/ChangeLog.epiphany
index 563a48f368b..0841c2a034a 100644
--- a/gcc/ChangeLog.epiphany
+++ b/gcc/ChangeLog.epiphany
@@ -1,3 +1,8 @@
+2019-03-24  Ola Jeppsson  <ola@adapteva.com>
+
+	* config/epiphany/epiphany.opt (fsoftware-cache): Add missing
+	dot in description. Fixes a testcase.
+
 2016-11-08  Ola Jeppsson  <ola@adapteva.com>
 
 	* config/epiphany/epiphany.md (define_constants)
diff --git a/gcc/config/epiphany/epiphany.opt b/gcc/config/epiphany/epiphany.opt
index be686df5fe9..c99d5299223 100644
--- a/gcc/config/epiphany/epiphany.opt
+++ b/gcc/config/epiphany/epiphany.opt
@@ -149,7 +149,7 @@ Enum(m1reg) String(r63) Value(63)
 
 fsoftware-cache
 Target Mask(SOFT_CACHE)
-Use software cache
+Use software cache.
 
 mno-sched-prolog
 Target RejectNegative Report Mask(NO_SCHED_PROLOG)
-- 
2.31.1

